cmd/skaffold/app/cmd/cmd.go

```diff
@@ -310,3 +308,28 @@ func preReleaseVersion(s string) bool {
 	}
 	return false
 }
+
+func isQuiet() (string, bool) {
+	switch {
+	case !interactive:
+		return "Update check prompt, survey prompt and telemetry disabled in non-interactive mode", true
```

Comment:
it's just the telemetry **prompt** we're disabling in these modes, not the actual telemetry itself right?

Reply:
Thanks. I checked if `ShouldDisplayMetricsPrompt` was only checking if prompt needs to be displayed. However, it also controls if metrics should be exported.

Maybe we need to separate this logic. Another task to do for another day.
I made changes to code. Will update soon.